io.github.letslearn_app.letslearn.yaml

```diff
@@ -0,0 +1,56 @@
+app-id: io.github.letslearn_app.letslearn
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '22.08'
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node16
+command: run.sh
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --device=dri
+  - --socket=x11
+  - --share=network
+build-options:
+  append-path: /usr/lib/sdk/node16/bin
+
+modules:
+  - name: letslearn
+    buildsystem: simple
+    build-options:
+      env:
+        XDG_CACHE_HOME: /run/build/letslearn/flatpak-node/cache
+        npm_config_cache: /run/build/letslearn/flatpak-node/npm-cache
+        npm_config_offline: 'true'
+    build-commands:
+      # Install npm dependencies
+      - npm install --offline
+      # Build the app; in this example the `dist` script
+      # in package.json runs electron-builder
+#      - |
+#        . ../flatpak-node/electron-builder-arch-args.sh
+      -  npm run build_electron -- $ELECTRON_BUILDER_ARCH_ARGS  --linux --dir --out=/run/build/letslearn/letslearn-linux-x64/ 
+      # Bundle app and dependencies
+      - cp -a /run/build/letslearn/letslearn-linux-x64/ /app/main
```

Comment:
this will fail on aarch64. (ie everything not x86_64)

Reply:
It wont, because output dir `/run/build/letslearn/letslearn-linux-x64/ ` has been set by `--out` so it will always be `/letslearn-linux-x64/` altought it is ugly.
By the way, how to force stop a build?

Reply:
then why do you set it to that? it's  misleading.

Reply:
because I forgot it 😂 i will change it now